sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -77,6 +78,20 @@ def test_retry_workflow(self):
     from .testdata.retry import retry_sample_pipeline
     self._test_pipeline_workflow(retry_sample_pipeline, 'retry.yaml')
 
+  def test_loop_static_workflow(self):
+    """
+    Test compiling a loop static params in workflow.
+    """
+    from .testdata.loop_static import pipeline
+    self._test_pipeline_workflow(pipeline, 'loop_static.yaml', normalize_compiler_output=lambda f: re.sub("loop-item-param-.*-subvar", "loop-item-param-subvar", f))
```

Comment:
break this line if it does not fit 120 character limit

Reply:
Updated, Thanks